apps/dialtone-documentation/docs/.vuepress/theme/assets/less/dialtone-docs.less

```diff
@@ -436,6 +436,13 @@ a.header-anchor {
   display: flex;
   width: 100%;
   height: 100%;
+
+  .d-popover {
+    height: 100%;
+    > div {
+      height: 100%;
+    }
+  }
```

Comment:
hmm do we need to override th d-popover class here? can we just apply this on the instance(s) where it's needed using class props and utility classes?

Reply:
I tried but couldn't, there is no class prop for the anchor and also passing a utility class to dt-popover doesn't apply the class